rest/route/select.go

```diff
@@ -47,6 +47,7 @@ func makeSelectTestsHandler() gimlet.RouteHandler {
 //	@Router			/select/tests [post]
 //	@Param			{object}	body	SelectTestsRequest	true	"Select tests request"
 //	@Security		Api-User || Api-Key
+//	@Success		200			{object}	model.APITest
```

Comment:
I don't think APITest is the correct return value here, but I needed to guess somehting :-)